services/motion/builtin/builtin_test.go

```diff
@@ -502,7 +385,7 @@ func TestObstacleReplanningSlam(t *testing.T) {
 		Destination:   spatialmath.NewPoseFromPoint(r3.Vector{X: 800, Y: 0, Z: 0}),
 		SlamName:      slam.Named("test_slam"),
 		MotionCfg: &motion.MotionConfiguration{
-			PositionPollingFreqHz: 1, ObstaclePollingFreqHz: 40, PlanDeviationMM: 1, ObstacleDetectors: obstacleDetectorSlice,
+			PositionPollingFreqHz: 0.000001, ObstaclePollingFreqHz: 5, PlanDeviationMM: 1000, ObstacleDetectors: obstacleDetectorSlice,
```

Comment:
You should be able to make the PositionPolling equal to 0 to turn it off right?

Reply:
Someday, but not today :)
https://viam.atlassian.net/browse/RSDK-7839